src/mpifx_init.F90

```diff
@@ -41,5 +41,52 @@ subroutine mpifx_init(error)
     call handle_errorflag(error0, "Error: mpi_init() in mpifx_init()", error)
 
   end subroutine mpifx_init
+
+
+  !> Initializes a threaded MPI environment.
+  !!
+  !! \param requiredThreading  Threading support required (MPI_THREAD_SINGLE, MPI_THREAD_FUNNELED,
```

Comment:
Worth adding either a description of the various levels, or a reference where to find this information?